README.md

```diff
@@ -1,14 +1,18 @@
-# Prefect Background Task Examples
+# Prefect Background Tasks
 
-This repository contains examples applications that demonstrate how to use Prefect to
-run background tasks. The examples include a FastAPI application that processes user
-signups and a Flask application that handles image uploads. Each example is
-self-contained, can be run locally, and includes automated tests.
+This repository contains example applications that demonstrate how to use [Prefect](https://prefect.io) to run background tasks.
+See the [Tutorial](./tutorial/) for step-by-step instructions showing progressively more advanced use cases.
 
-## Why Prefect for Background Tasks?
+## Why use Prefect background tasks?
 
-If you are familiar with Python background task tools like Celery, RQ, and arq, Prefect offers a similar Pythonic
-interface for defining and running tasks, paired with a robust set of features:
+Prefect background tasks are a great way to quickly execute discrete units of work in background processes.
+For example, if you have a web app, you can use Prefect background tasks to offload processes such as sending emails, processing images, or inserting data into a database.
+Prefect background tasks are ideal for a microservices architecture, where you have a number of small, independent services that need to communicate with each other.
+As we'll see, Prefect background tasks make it simple to parallelize workloads, cache results, and manage retries and timeouts.
```

Comment:
"Results" is product language. Let's speak to users who haven't used Prefect before.

```suggestion
With Prefect, you can run tasks in parallel, cache return values, configure automatic retries, and more, with a simple interface that scales up to complex workflows that need dynamic infrastructure.
```